src/python/pants/jvm/package/war.py

```diff
@@ -0,0 +1,173 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import logging
+import textwrap
+from dataclasses import dataclass
+from pathlib import PurePath
+
+from pants.base.glob_match_error_behavior import GlobMatchErrorBehavior
+from pants.build_graph.address import Address
+from pants.core.goals.package import (
+    BuiltPackage,
+    BuiltPackageArtifact,
+    OutputPathField,
+    PackageFieldSet,
+)
+from pants.core.target_types import ResourceSourceField, FileSourceField
+from pants.core.util_rules.archive import ZipBinary, UnzipBinary
```

Comment:
I can't see where `UnzipBinary` is being used here. Is it necessary?

Reply:
Removed.